Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -3384,6 +3384,44 @@ void WebPage::updateFrameSize(WebCore::FrameIdentifier frameID, WebCore::IntSize
     }
 }
 
+void WebPage::frameCompositedBoundsChanged(WebCore::FrameIdentifier frameID, WebCore::IntPoint contentsOffset)
+{
+    if (!m_page)
+        return;
+
+    ASSERT(m_page->settings().siteIsolationEnabled());
+    RefPtr webFrame = WebProcess::singleton().webFrame(frameID);
+    if (!webFrame)
+        return;
+
+    RefPtr frame = webFrame->coreLocalFrame();
```

Comment:
This is another hard to see WebProcess -> WebProcess through UIProcess IPC.

Reply:
Unfortunately there will be many many more :(